includes/resourceloader/PortableInfoboxResourceLoaderModule.php

```diff
@@ -0,0 +1,10 @@
+<?php
+
+class PortableInfoboxResourceLoaderModule extends ResourceLoaderFileModule {
+	/** @inheritDoc */
+	protected function getLessVars( ResourceLoaderContext $context ) {
+		$lessVars = parent::getLessVars( $context );
+		$lessVars[ 'responsibly-open-collapsed'] = $this->getConfig()->get( 'PortableInfoboxResponsiblyOpenCollapsed' );
```

Comment:
Would forcing this to a (bool) be good here, or unnecessary? I'm not 100% sure either way.

Reply:
`(bool)` prepended. I am not sure too but think making it safe is good.

Reply:
Thank you!